File: My Solutions/Python/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ht = {}
        for i, num in enumerate(nums):
            n = target - num
            if n not in ht:
                ht[num] = i
            else:
                return [ht[n], i]

File: My Solutions/PHP/twoSum.php
<?php

class Solution {

    /**
     * @param Int[] $nums
     * @param Int $target
     * @return Int[]
     */
    function twoSum($nums, $target) {
        for ($x = 0; $x < count($nums); $x++) {
            for ($j = $x+1; $j < count($nums); $j++) {
                if($nums[$x] + $nums[$j] === $target) {
                    return [$x, $j];
                }
            }
        }
    }
}